SteamList/Views/NewsTableViewCell.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  NewsTableViewCell.swift
+//  SteamList
+//
+//  Created by Macbook on 20/01/2023.
+//
+
+import UIKit
+
+class NewsTableViewCell: UITableViewCell {
+    
+    static let identifier = "NewsTableViewCell"
+    
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .white
+        label.font = .boldSystemFont(ofSize: 18)
+        
+        return label
+    }()
+    
+    private let gameNameLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .white
+        label.font = .systemFont(ofSize: 16, weight: .light)
+        
+        return label
+    }()
+    
+    private let developerLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .white
+        label.font = .italicSystemFont(ofSize: 16)
+        
+        return label
+    }()
+    
+    private let dateLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .white
+        label.font = .systemFont(ofSize: 16, weight: .light)
+        
+        return label
+    }()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        contentView.addSubview(titleLabel)
```

Comment:
you could extract all those changes to a method `setupContentView`

Reply:
Agree, but `func constructHierarchy()` would sound better